tools/acrn-crashlog/README.rst

```diff
@@ -0,0 +1,124 @@
+ACRN-Crashlog
+#############
+
+Introduction
+************
+
+The ``ACRN-Crashlog`` is a debug tool to collect any useful logs/information after each crash/event occurred on the platform, including hypervisor, Service OS(SOS) and Android as a Guest(AaaG). It provides abundant logs for developers to debug/analyze the crashes. And it provides a flexible way to configure which crash/event to be concerned with xml file.
+
+Building
+********
+
+Build dependencies
+==================
+
+The ``ACRN-Crashlog`` tool depends on the following libraries (build and runtime):
+
+- libevent
+
+- OpenSSL
+
+- libxml2
+
+- systemd
+
+- telemetrics-client-dev (optional, detected at build time)
+
+For more detailed instructions on how to set-up your build environment, please refer to the :ref:`getting_started`.
+
+Build
+=====
+
+To build the ``ACRN-Crashlog``
```

Comment:
You should specify what directory you need to be in before you do the ``make`` command

Reply:
Thanks for your suggestion, and the directory will be showed in the next version.